examples/quickstart/20-pod.yaml

```diff
@@ -55,9 +53,6 @@ spec:
         name: sonobuoy-plugins-cm
     # NOTE: This will drop the results on the hostpath of the machine as an example.
     # We recommend using a persistent volume for more permanent recording.
-    - name: output-volume
-      hostPath:
```

Comment:
@kensimon it's a remnant from before you and Tim added `no-exit`. Previously, when the pod finished it disappeared I think (so the `emptyDir` would be removed with it?), so I changed the example to `hostPath` so that I could SSH in afterwards and poke at it.

TLDR should be fine to change to emptyDir. If you do so, could you also take out one of the sub-bullet points in https://github.com/heptio/sonobuoy/blob/master/docs/configuration.md#pod (the part under "Output Volume Mount")?